packages/core/lib/generators/integrations/image_integration.dart

```diff
@@ -11,9 +12,12 @@ import 'package:image_size_getter/image_size_getter.dart';
 class ImageIntegration extends Integration {
   ImageIntegration(
     String packageName, {
+    required this.parseAnimation,
```

Comment:
If the option is for the image only, the option definition might fall into an image integration section rather than at the same level as `parse_metadata` in yaml.

Reply:
Move the `parse_animation` option to a dedicated `images` section. You can see changes in README for details.